cloudsync/tasks.py

```diff
@@ -95,14 +135,37 @@ def transcode_from_s3(self, video_id):  # pylint: disable=unused-argument
 
     # Generate thumbnails for the 1st encoding (no point in doing so for each).
     outputs[0]['ThumbnailPattern'] = THUMBNAIL_PATTERN.format(video_file.s3_basename)
-    transcoder = Transcoder(
+    transcoder = VideoTranscoder(
         settings.ET_PIPELINE_ID,
         settings.AWS_REGION,
         settings.AWS_ACCESS_KEY_ID,
         settings.AWS_SECRET_ACCESS_KEY
     )
-    transcoder.encode(video_input, outputs, Playlists=playlists)
-    transcoder.create_job_for_object(video)
+    try:
+        if not settings.ET_PRESET_IDS:
+            raise ValueError("At least one transcode preset required in settings")
+        transcoder.encode(video_input, outputs, Playlists=playlists)
+    except (ValueError, ClientError) as e:
```

Comment:
nit: if you are using tuples, please end the "list" with a comma to make clear that it is a tuple
`(ValueError, ClientError,)`